### ISSUE TITLE
Secured SQLite wrong password

### ISSUE THREAD

**Issue by rumplestilzken** [giuspen/cherrytree#37]
i have a secured sqlite based cherrytree document. When i tried to open it on another computer that has .35 and it told me wrong password. When i installed .32 (The same as the computer that created the document and was able to open fine with password) i was able to open the document.

**Comment by giuspen** [comment]
Hi, there is a known bug that should be already fixed in the version I'm
going to release in the next days.
You can test the fix downloading the release candidate from
http://www.giuspen.com/topic/cherrytree-0-35-7r1590-upcoming-0-35-8/

**Comment by DaniyalYaseen-ai** [comment]
Dear CherryTeam and Community,

I am using Cherrytree 39.2 from a time long and the file I am using is encrypted and now its been prompted for the wrong password every time I am trying to access it. The password is 100% current and I even checked opening the file on new updated stable Cherrytree version 39.4 but the same wrong password is been shown. Please help me out.

**Comment by giuspen** [comment]
@DaniyalYaseen-ai try to open the password protected document directly with https://www.7-zip.org/ - if there is no way to open it try the backups that are in the same folder, have the same name of the document but trailing tildes (e.g. file.ctx~, file.ctx~~)

**Comment by DaniyalYaseen-ai** [comment]
@giuspen  None of your suggested method worked.

1) With the Zip >> It says unable to open with archive 
2) file.ctx~ etc >> shows nothing but just an empty new Cherrytree file

@giuspen I am now tied of it please help

**Comment by DaniyalYaseen-ai** [comment]
@giuspen  None of your suggested method worked.

1) With the Zip >> It says unable to open with archive 
2) file.ctx~ etc >> shows nothing but just an empty new Cherrytree file
@giuspen I am now tied of it please help

**Comment by giuspen** [comment]
Try remove the tildes before opening. Have you recently changed password? If yes and introduced special characters let me know what characters because we had an issue with a special character where the password is just different than what you expect for the character (s) from the special character onwards

**Comment by DaniyalYaseen-ai** [comment]
The file I am not able to open was made a year back with the password set once as of now, I have not changed or updated the password and my password for this file is 12345@cherrytree

You  may notice that I have used simple and single special character that is "@"

**Comment by giuspen** [comment]
There is no issue with that character, it must be something else went wrong in the encryption process. We are about to release a new totally rewritten and way more robust version of cherrytree https://www.giuspen.com/cherrytree/#testing but I'm afraid there's no way to recover that document

**Comment by DaniyalYaseen-ai** [comment]
@giuspen  What?  This means there is a possibility of not recovering the file? I have my data of around 1 GB inside that file compromising my backups, projects information. I will be completely destroyed If not able to recover that file.

**Comment by giuspen** [comment]
So if your computer disk dies your life is destroyed?

**Comment by giuspen** [comment]
How can you have 1GB of data in a password protected archive, how long it took to save each time, minutes I guess... that is really pushing the boundaries of this free application

**Comment by DaniyalYaseen-ai** [comment]
Yes, I know its a free application but due to its popularity, many of us use it for daily taking notes etc. Even my colleagues in the office used it to store critical information.

The word Destroyed means that I will be in a huge problem.

**Comment by giuspen** [comment]
I'm really sorry when these things happen, in the past I was even wondering if it was right to deliver my free time work when users happened to lose data with it but  in the end I thought about the wide majority of users that are happy with it, so I do my best to make it better, try to reduce the possibility that it happens again, but cannot zero that possibility unfortunately, for critical data there is no other safety than a time based backup on a different disk

**Comment by CyberPhobicGeek** [comment]
hey @DaniyalYaseen-ai I got a different result which may help you. I have had the same exact issue you had. I had multiple backups: NoteBook.ctx, NoteBook.ctx~, NoteBook.ctx~~, NoteBook.ctx~~~.
I tried opening NoteBook.ctx and it failed.
I treid opening NoteBook.ctx~ and it failed.
I tried opening NoteBook.ctx~~ and voila I managed to place my password and got to the NoteBook.ctb version. From there I just made a copy and opened it again. In my case, the file wasnt big, nor was did the password have any special characters at all.

(All this is in windows 10, I am assuming that after I did my windows virus scanner the issue happened..)
